src/attributes/classes.js

```diff
@@ -23,9 +39,7 @@ jQuery.fn.extend( {
 			} );
 		}
 
-		if ( typeof value === "string" && value ) {
-			classes = value.match( rnothtmlwhite ) || [];
```

Comment:
@faisaliyk The problem is that the committed files include merge conflict markers. For example see [this line](https://github.com/jquery/jquery/pull/3727/commits/4e62134dc6309293152ec9aef836038ff83eae1f). Perhaps you didn't re-push to the PR branch after resolving the merge conflicts?